src/common/imagwebp.cpp

```diff
@@ -0,0 +1,346 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        src/common/imagwebp.cpp
+// Purpose:     wxImage WebP handler
+// Author:      Hermann Höhne
+// Copyright:   (c) Hermann Höhne
+// Licence:     wxWidgets licence
+/////////////////////////////////////////////////////////////////////////////
+
+// based on code by Sylvain Bougnoux, Khral Steelforge and Markus Juergens
+// see https://forums.wxwidgets.org/viewtopic.php?t=39212
+
+// For compilers that support precompilation, includes "wx.h".
+#include "wx/wxprec.h"
+
+#if wxUSE_IMAGE && wxUSE_LIBWEBP
+
+#include "wx/imagwebp.h"
+#include "webp/demux.h"
+#include "webp/decode.h"
+#include "webp/encode.h"
```

Comment:
Those should use `<...>` include style.

And I'd also add blank lines before and after them as they're a different group of headers.